### DISCUSSION TITLE
Problem on FundMe Contract

### DISCUSSION THREAD

**Discussion by Zyrrow** [Cyfrin/foundry-full-course-cu#2235] [Q&A] [answered]
Hi guys 
I currently have a problem with my function getPrice and getConversionRate. Here is my error :
![error-solidity](https://github.com/user-attachments/assets/9d531225-3c9e-4030-93cb-f34852b13a58)

I already tried to increase gas limit, checked multpile time the pricefeed address. I also ask AI but nothing is working. Here is my code :

![fundme](https://github.com/user-attachments/assets/4f78ee90-b3c3-4caa-9603-40a01a69433e)

Please help

**Comment by Maa-ly** [comment]
The issue here is your conversionRate. It cant seem to get the correct ETh to USA rate. Thats why i thinking youre getting the error.  You should use the latest Eth to Usa conversionRate.

**Reply by Zyrrow** [reply]
I added a getVersion function and it still doesn't work, i have the feeling that every function with the eth/usd sepolia address will fall into error. I don't understand it because im using the address on https://docs.chain.link/data-feeds/price-feeds/addresses?network=ethereum&page=2

**Comment by EngrPips** [comment]
Hello @Keeyki, what chain are you running on?

**Reply by Zyrrow** [reply]
Im running on remix with false eth

**Reply by EngrPips** [reply, marked as answer]
If you connect the remix to sepolia chain, the contract will work. It won't work because the pricefeed address you are using in your `getPrice` function is for only for sepolia chain.

**Reply by Zyrrow** [reply]
Oh ok i understand thank you mate !

**Reply by EngrPips** [reply]
You are welcome, Friend. Keep crushing it.
